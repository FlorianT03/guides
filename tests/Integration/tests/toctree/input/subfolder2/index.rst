===============
Subfolder Index
===============

A Toctree with multiple whitespaces in the directive:

..  toctree::
    subpage1
    subpage2
