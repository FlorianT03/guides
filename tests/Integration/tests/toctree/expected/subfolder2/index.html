<!DOCTYPE html>
<html lang="en">
<head>

</head>
<body>
    <div class="section" id="subfolder-index">
        <h1>Subfolder Index</h1>
        <p>A Toctree with multiple whitespaces in the directive:</p>

            <div class="toc"><ul class="phpdocumentor-list"><li class="toc-item"><a href="/subfolder2/subpage1.html#subpage-1">Subpage 1</a></li><li class="toc-item"><a href="/subfolder2/subpage2.html#subpage-2">Subpage 2</a></li></ul></div>
    </div>

</body>
</html>
